aiida/restapi/translator/node.py

```diff
@@ -604,20 +606,21 @@ def get_node_shape(ntype):
                     "to": 0,
                     "arrows": "to",
                     "color": {"inherit": 'from'},
-                    "linktype": linktype,
+                    "label": linktype,
```

Comment:
The linklabel should be moved here from `nodes`, and this should remain to be called linktype

Reply:
'label' is the name used by JS library to display value on edge. This endpoint is specifically written for materials cloud and passes the data in format used in MC. Otherwise we can reprocess the data completely in frontend again before passing it to the visualization library.